logever/src/main/java/ai/smartfac/logever/entity/User.java

```diff
@@ -26,19 +26,30 @@ public class User implements UserDetails {
     private String first_name;
     private String last_name;
     private String password;
+    private String department;
     @Transient
     @JsonIgnore
     private Date create_dt;
     @Transient
     @JsonIgnore
     private Date update_dt;
 
-    @JsonIgnore
-    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @ManyToMany( fetch = FetchType.EAGER)
```

Comment:
Correct, but the API to get users can be called some other times in future. Usually it is best practice to use FetchType.LAZY for Many-Many relationship. It will fetch values from relationship only when getRoles methods is called.

Reply:
Done